couch/tests/conftest.py

```diff
@@ -84,82 +115,83 @@ def send_replication(couch_version):
         'create_target': True,
         'continuous': True,
     }
-    r = requests.post(
-        replicator_url,
-        auth=(common.NODE1['user'], common.NODE1['password']),
-        headers={'Content-Type': 'application/json'},
-        json=replication_body,
-    )
-    r.raise_for_status()
-
-
-def get_replication(couch_version):
+    for i in range(5):
+        print("[INFO] create replication task {}".format(i))
+        body = replication_body.copy()
+        body['_id'] = 'my_replication_id_{}'.format(i)
+        body['target'] = body['target'] + str(i)
+        r = requests.post(
+            replicator_url,
+            auth=(common.NODE1['user'], common.NODE1['password']),
+            headers={'Content-Type': 'application/json'},
+            json=body,
+        )
+        r.raise_for_status()
+        print("[INFO] replication task created:", r.json())
+
+
+def get_replication():
     """
     Attempt to get active replication tasks
     """
-    if couch_version == '1':
-        return
-
     task_url = "{}/_active_tasks".format(common.NODE1['server'])
 
     r = requests.get(task_url, auth=(common.NODE1['user'], common.NODE1['password']))
     r.raise_for_status()
-    count = len(r.json())
+    active_tasks = r.json()
+    print("[INFO] active_tasks response:", active_tasks)
+    count = len(active_tasks)
     return count > 0
 
 
-def generate_data(couch_version):
-    """
-    Generate data on the couch cluster to test metrics.
-    """
-    # pass in authentication info for version 2
-    auth = (common.USER, common.PASSWORD) if couch_version == "2" else None
+def enable_cluster():
```

Comment:
(Nit) Consider moving this above `generate_data()` to follow the order of `conditions`?

Reply:
@florimondmanca Sure. Good idea. Reordered :)